### ISSUE TITLE
Improve color contrast in Platform

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4180]
There must be a color contrast ratio of at least 4.5:1 between all text and background. Audit and suggest a new color-scheme where needed (needs design).

Places where contrast-ratio fails:
Between background and button:
<img width="275" alt="Screenshot 2020-12-11 at 11 14 15" src="https://user-images.githubusercontent.com/8624777/101891312-2b2c5b00-3ba2-11eb-9905-437cd98d14fe.png">

Between background and number:
<img width="189" alt="Screenshot 2020-12-11 at 11 17 04" src="https://user-images.githubusercontent.com/8624777/101891511-6a5aac00-3ba2-11eb-9640-88c5c749d252.png">

Between background and text:
<img width="265" alt="Screenshot 2020-12-11 at 11 18 38" src="https://user-images.githubusercontent.com/8624777/101891657-a1c95880-3ba2-11eb-8c57-d2b42c6a9545.png">

Between background and toggle-button:

<img width="147" alt="Screenshot 2020-12-11 at 11 20 49" src="https://user-images.githubusercontent.com/8624777/101891885-ef45c580-3ba2-11eb-86fe-16ba26b64463.png">

Between the background and the text:
<img width="137" alt="Screenshot 2020-12-11 at 11 32 17" src="https://user-images.githubusercontent.com/8624777/101893086-88c1a700-3ba4-11eb-9fa3-6db6cd3b12b7.png">

Between background and text:
<img width="449" alt="Screenshot 2020-12-11 at 11 33 44" src="https://user-images.githubusercontent.com/8624777/101893204-bd356300-3ba4-11eb-9633-bfecfde498c9.png">

Link to W3:
https://www.w3.org/TR/WCAG21/#contrast-minimum
